signingscript/src/signingscript/utils.py

```diff
@@ -238,3 +275,22 @@ def split_autograph_format(format_):
         return format_.split(":", 1)
     else:
         return format_, None
+
+
+async def extract_dmg(context, source, dest):
+    """Extracts a source DMG into dest folder
+
+    Args:
+        context: The running task context object
+        source (str): the DMG sorce path
+        dest (str): the destination directory to extract files
+    """
+    dmg = context.config["dmg"]
+    hfsplus = context.config["hfsplus"]
+    undmg_cmd = [dmg, "extract", source, "tmp.hfs"]
+    log.info(undmg_cmd)
+    await execute_subprocess(undmg_cmd, cwd=context.config["work_dir"], log_level=logging.DEBUG)
```

Comment:
question: Did we ever decide what to do about the symlinks? If we're not handling them here, have you tested to make sure the outputs from `repackage-signing` still contain working symlinks?

Reply:
I was testing this in adhoc signer, which doesn't have repackaging setup.
I guess we could try this on Try?

Reply:
That sounds like a good next step!

Reply:
Did this testing happen?